### DISCUSSION TITLE
Eglot cant find an LSP for emacs-lisp-mode?

### DISCUSSION THREAD

**Discussion by refaelsh** [joaotavora/eglot#1212] [Q&A] [answered]
I installed eglot. Closed Emacs. Reopened it on my Emacs config file.
Did M-x eglot RET. Got the following error:
```
Could not guess LSP server for `emacs-lisp-mode`
```
Here comes a dump question: I understand that Eglot can't guess every LSP out there, but Elisp? It can't guess Elisp? Its like the shoe maker that walks without shoes.

**Comment by joaotavora** [comment, marked as answer]
A bit yes. :)

But Emacs is a (half-decent) lisp machine (look it up ;-) ), a think from the 80's that is about one trillion times better than LSP.

Iow, i can't think of many LSP features that aren't better implemented in Emacs already. Can you?

**Reply by refaelsh** [reply]
I understand. Thank you for the answer.

**Reply by stormwatch** [reply]
We should definitely work on an LSP server for emacs lisp and disclose it next April 1st.
